web-app/HeimScripts/heatmap/run.R

```diff
@@ -176,3 +189,85 @@ add.subset.label <- function(df,label){
   }
   return(df)
 }
+
+computeDendrogram <- function(zScoreMatrix, distanceMeassure, linkageMethod) {
+    as.dendrogram(hclust(dist(zScoreMatrix, method=distanceMeassure), method=linkageMethod))
+}
```

Comment:
You mean that it does not matter if you select single or complete for euclidean distance?